tests/e2e/objects/FileExplorer.ts

```diff
@@ -0,0 +1,33 @@
+import type { Locator, Page } from "@playwright/test";
+import { expect } from "@playwright/test";
+import { ActivityBarItem } from "./ActivityBarItem";
+import { View } from "./views/View";
+
+/**
+ * Object representing the VS Code File Explorer view.
+ * Provides methods to interact with files and folders in the explorer.
+ */
+export class FileExplorer extends View {
+  constructor(page: Page) {
+    super(page, "Explorer");
+  }
+
+  /** Override to use the more specific Explorer view ID. */
+  override get locator(): Locator {
+    return this.page.locator('[id="workbench.view.explorer"]');
+  }
+
+  /**
+   * Ensure the Explorer view is visible.
+   */
+  async ensureVisible(): Promise<void> {
+    try {
+      await expect(this.locator).toBeVisible({ timeout: 2000 });
+    } catch {
+      const activityBarItem = new ActivityBarItem(this.page, "Explorer");
+      await expect(activityBarItem.locator).toBeVisible();
+      await activityBarItem.locator.click();
+      await expect(this.locator).toBeVisible();
+    }
+  }
```

Comment:
Future: this along with https://github.com/confluentinc/vscode/blob/main/tests/e2e/utils/sidebarNavigation.ts could probably stand to be repurposed as general helper methods within the [`ViewContainer` page object](https://github.com/confluentinc/vscode/blob/main/tests/e2e/objects/ViewContainer.ts) model so we can easily check if the container is open/visible, and only then associate it with an icon in the activity bar. That way tests/utils/POMs can check the view container before working with any individual `View` (or subclass) instances